.github/workflows/pr-review-checklist.yml

```
@@ -0,0 +1,21 @@
+
+on:
+  pull_request:
+    types: 
+      - opened
+
+jobs:
+  pr-review-list:
+    name: Post PR Review Checklist
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)